build/_build/Build.Steps.cs

```diff
@@ -37,6 +37,7 @@ partial class Build
     AbsolutePath ArtifactsDirectory => Artifacts ?? (OutputDirectory / "artifacts");
     AbsolutePath WindowsTracerHomeZip => ArtifactsDirectory / "windows-tracer-home.zip";
     AbsolutePath BuildDataDirectory => RootDirectory / "build_data";
+    AbsolutePath LibSqreenDirectory => RootDirectory / "packages" / "libsqreen.1.1.2.2";
```

Comment:
Assuming that this is pulled in using a normal nuget restore, we should probably use something like this:
```suggestion
    AbsolutePath LibSqreenDirectory => (NugetPackageDirectory ?? (RootDirectory / "packages")) / "libsqreen.1.1.2.2";
```